distribution/docs/src/main/resources/content/_integrating/_endpoints/query-endpoints.adoc

```diff
@@ -0,0 +1,241 @@
+:title: Queries Endpoint
+:type: endpoint
+:status: published
+:operations: crud
+:link: _queries_endpoint
+:summary: To perform CRUD (Create, Read, Update, Delete) operations on query metacards in the catalog, work with one of these endpoints.
+
+== {title}
+
+The queries endpoint enables an application to create, retrieve, update, and delete query metacards.
```

Comment:
Should we define query metacards here? As to how they are different from other metacards?